src/shared-components/immersiveModal/index.tsx

```diff
@@ -180,49 +179,17 @@ export const ImmersiveModal: ImmersiveModal = ({
                 className="modal-offclick-wrapper"
               >
                 <Style.MobileTopOverlay onClick={handleCloseIntent} />
-                <FocusScope contain restoreFocus>
-                  <Style.MainModalContentContainer
-                    id={MODAL_DESKTOP_SCROLLING_ID}
-                    hasHeaderImage={hasHeaderImage}
+                <FocusScope contain restoreFocus autoFocus>
+                  <ImmersiveModalContent
+                    footerContent={footerContent}
+                    headerImage={headerImage}
+                    showDesktopHeaderBar={showDesktopHeaderBar}
+                    showMobileHeaderBar={showMobileHeaderBar}
+                    title={title}
+                    handleCloseIntent={handleCloseIntent}
                   >
-                    <Style.CrossIconContainer
-                      onClick={handleCloseIntent}
-                      tabIndex={showDesktopHeaderBar ? -1 : 0}
```

Comment:
Because there is only one `CrossIconButton` now (formerly named `CrossIconContainer`) we can remove the `tabIndex` handling.